src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -38,7 +38,7 @@ public void testIsFull() {
         }
 
         boolean result = rhymer.isFull();
-        Assert.assertEquals(true, result);
+        Assert.assertEquals(false, result);
```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project. If STACK_CAPACITY is 12 then 12 elements should be added to the stack.